fix some inconsistency in import aliases

thaJeztah opened the pull request:
**- A picture of a cute animal (not mandatory but encouraged)**

thaJeztah committed:
fix some inconsistency in import aliases

Signed-off-by: Sebastiaan van Stijn <github@gone.nl>

akerouanton reviewed (commented):
Is it worth adding an `importas` alias in `.golangci.yml`?

thaJeztah commented:
> Is it worth adding an `importas` alias in `.golangci.yml`?

Yeah, perhaps indeed. I was planning to, then considered that perhaps this package should be moved internal and renamed, but it's still shared between "builder" and "daemon", so we can't move it yet.

I added a rule; will merge this one when validation completes.
